### DISCUSSION TITLE
Feature Request: Support for "of:accepts:xmtp" Metadata Property

### DISCUSSION THREAD

**Discussion by 0x94t3z** [wevm/frog-archived#171] [Ideas]
## Feature Description

I'm proposing the addition of a new metadata property, specifically `of:accepts:xmtp`, to enhance the capabilities of Frog with XMTP protocol support. This property would specify the version or date until which the protocol is accepted, providing clearer communication and compatibility expectations between clients.

## Current Behavior

Currently, Frog does not explicitly support or recognize the `of:accepts:xmtp` property in its metadata handling. This limits the framework's ability to interact with projects that use XMTP or similar protocols requiring version control and forward compatibility.

## Example Metadata

1. **HTML Meta Tags**:

   ```html
   <meta property="of:accepts:xmtp" content="2024-02-01" />
   ```
   
2. **OnChainKit Metadata**:

    ```typescript
    export const metadata: Metadata = {
      title: "Your frame metadata",
      description: "Your frame metadata",
      openGraph: {/* Your frame metadata */},
      other: {
        ...frameMetadata,
        'of:accepts:xmtp': '2024-02-01',
      },
    };
    ```
    
3. **Frog Metadata?**

**Comment by dalechyn** [comment]
https://frog.fm/concepts/middleware#xmtp-frames-middleware
